### ISSUE TITLE
Two Splash Screens on Android 13+ after releasing via Google Play

### ISSUE THREAD

**Issue by kashirin-dm** [FlutterFlow/flutterflow-issues#2598]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

I have Splash Screen added inside FlutterFlow.
In debug mode, I see the Flutter loading screen and then my Splash Screen. I assume this is normal behavior.
When I release via Google Play and then launch the app, I first see the Splash Screen from Android 13 (solid background, square with rounded edges, and the app icon in the middle), and then I see my Splash Screen.

On Android below version 13, like 12 or 9, I only see the Splash Screen added inside FlutterFlow.

### Expected Behavior

Only Splash Screen added via FlutterFlow on any version of Android should be displayed.

### Steps to Reproduce

1. Add a Splash Screen image inside FlutterFlow.
2. Make an internal release on Google Play.
3. Download the app via Play Market to a device with Android 13 or higher and run it.

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4shcj18ZdPxOpE0LruY8dB/T86JVN+aq8ascsZbw4bB4zqPLJ/esjObhFCcMSrd2lhHFWLsEgCpdrwkfL1UMYoMgqsfr5GwZZAUQHyZ3q8Sq6PPM2RdkZBN8BQJhWb0LSvuwp7Ju1YcWQZwU+QfPCidCvsN5+/Zwh9f6PHaOI=

### Context

I get notifications from Google Play Console; it slows down the app loading, can't be customized, and doesn't look aesthetically pleasing. I'm unsure if the app will pass the review with two Splash Screens.

### Visual documentation

Strange Splash Screen on Android 13
![Screenshot 2024-03-26 182829](https://github.com/FlutterFlow/flutterflow-issues/assets/40428935/4a849bad-67ad-45ce-9790-47aecece502c)

### Additional Info

I can add you to the internal tester list so you can run the app and take a look.

### Environment

```markdown
- FlutterFlow version: 1.8.10
- Platform: Android 13, 14
```

**Comment by bozzoleandro** [comment]
Hi @kashirin-dm, Thanks for your report, did you check the documentation for adding Splash screens from version 12 onwards? [https://developer.android.com/develop/ui/views/launch/splash-screen?hl=es-419](https://developer.android.com/develop/ui/views/launch/splash-screen?hl=es-419), [https://docs.flutter.dev/platform-integration/android/splash-screen](https://docs.flutter.dev/platform-integration/android/splash-screen). Please check and let us know if that works for you.

**Comment by kashirin-dm** [comment]
Yeah, I read it. Apparently, improvements are needed in the FlutterFlow interface to allow Splash Screen editing for different versions of Android and to prevent the old Splash Screen from being shown on newer versions of Android.
I can make these edits via code, but why would I need FF?

**Comment by leighajarett** [comment]
We've confirmed this issue on our end and will work on a fix as soon as possible!

**Comment by sanjayelate** [comment]
It's now august, is there any update on this yet?

**Comment by kashirin-dm** [comment]
> It's now august, is there any update on this yet?

I did a release on Google Play. The problem may still be, but the app looks fine.

**Comment by biteam2024** [comment]
This issue is preventing us from publishing to Google Play—there's a double splash screen, and it's getting rejected by Google.  

![Image](https://github.com/user-attachments/assets/210faa7e-223b-41e7-bb05-6e9058b9260b)

**Comment by abdallahabbas20** [comment]
Lots of flutterflow forums full of people suffering this issue. @leighajarett Please could an update be provided as to whether this is being fixed as it is stopped us from publishing to Google Play + leading to very bad UI experience. Thank you so much!

**Comment by Wojtek-G** [comment]
I have same issue.

**Comment by Bolt8743** [comment]
I also have Google reporting this. I have just re-submitted my application for production and did not notice this until after submission. My initial application was rejected for a different reason but based on what is reported here, I expect that this one will also be rejected. My previous submission was not getting this warning and I have not made any changes to the splash screen. At a minimum, it looks like users will need to ha e an option to disable the splash screen for android just like the option exists for web in General -> Settings -> App Assets -> Splash. More full featured resolution would be to provide options to FF users to enter values that are passed to the flutter_native_splash package api that allow for setting the options for android 12 and higher.

At this point, it looks like FF users will need to not set a splash image in App Assets and likely use a different package if issues arise with submitting apps to Apple and Google. This however defeats the purpose of FF for many users that do not have a coding background.

**Comment by LeCause** [comment]
Same issue on my side.
Play store warning concerning the double splash screen and a respective user experience.
Looking forwar to a soon fix of the issue at is open for a year, now.
Thanks in advance!

**Comment by ALaiFF** [comment]
Same issue

**Comment by czsday** [comment]
same issue
